Error brought up by schedule unit testing experiments

Trying to run schedule_modifier.py for CLY, and get the following error partway through the experiment unit testing:

```
Running tests on experiments ['full_fov_15km']
Traceback (most recent call last):
  File "/home/radman/borealis/scheduler/schedule_modifier.py", line 54, in <module>
    main()
  File "/home/radman/borealis/scheduler/schedule_modifier.py", line 45, in main
    scd_util.add_line(args.date, args.time, args.experiment, args.mode_type, args.prio, args.duration, kwargs,
  File "/home/radman/borealis/scheduler/scd_utils.py", line 235, in add_line
    scd_lines = self.read_scd()
                ^^^^^^^^^^^^^^^
  File "/home/radman/borealis/scheduler/scd_utils.py", line 158, in read_scd
    scd_lines.append(self.check_line(line[0], line[1], line[4], line[5], line[3], line[2], kwargs,
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/radman/borealis/scheduler/scd_utils.py", line 120, in check_line
    raise ValueError("Experiment could not be scheduled due to errors in experiment.\n"
ValueError: Experiment could not be scheduled due to errors in experiment.
Errors: []
Failures: [(<experiment_unittests.TestActiveExperiments testMethod=test_full_fov_15km>, 'Traceback (most recent call last):\n  File "/home/radman/borealis//tests/experiments/experiment_unittests.py", line 276, in test\n    redirect_to_devnull(ehmain, experiment_name=module_name, **kwargs)\n  File "/home/radman/borealis//tests/experiments/experiment_unittests.py", line 49, in redirect_to_devnull\n    result = func(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^\n  File "/home/radman/borealis//tests/experiments/experiment_unittests.py", line 72, in ehmain\n    exp = experiment(**kwargs)\n          ^^^^^^^^^^^^^^^^^^^^\n  File "/home/radman/borealis//src/borealis_experiments/full_fov_15km.py", line 79, in __init__\n    self.add_slice({  # slice_id = 0, there is only one slice.\n  File "/home/radman/borealis//src/experiment_prototype/experiment_prototype.py", line 799, in add_slice\n    new_exp_slice = ExperimentSlice(**exp_slice, **self.__slice_restrictions)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "pydantic/dataclasses.py", line 325, in pydantic.dataclasses._add_pydantic_validation_attributes.new_init\n  File "pydantic/dataclasses.py", line 425, in pydantic.dataclasses._dataclass_validate_values\npydantic.error_wrappers.ValidationError: 1 validation error for ExperimentSlice\nnum_ranges\n  ensure this value is less than or equal to 100 (type=value_error.number.not_le; limit_value=100)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/radman/borealis//tests/experiments/experiment_unittests.py", line 278, in test\n    self.fail(err)\nAssertionError: 1 validation error for ExperimentSlice\nnum_ranges\n  ensure this value is less than or equal to 100 (type=value_error.number.not_le; limit_value=100)\n\nStdout:\n\nExperiment Test:  test_full_fov_15km\n')]
```

It seems to be complaining about num_ranges being greater than 100, which it was for this experiment (it was set to 225). For 15km experiments or other non-standard configurations, there very well could be reason to run 100+ range gates. This should be amended in ExperimentSlice.

--- a/src/experiment_prototype/experiment_slice.py
+++ b/src/experiment_prototype/experiment_slice.py
@@ -277,7 +277,7 @@ class ExperimentSlice:
     beam_angle: conlist(Union[confloat(strict=True), conint(strict=True)], unique_items=True)
     cpid: StrictInt
     first_range: Union[confloat(ge=0), conint(ge=0)]
-    num_ranges: conint(gt=0, le=options.max_range_gates, strict=True)
+    num_ranges: conint(gt=0, strict=True)
     tau_spacing: conint(ge=options.min_tau_spacing_length, strict=True)
     pulse_len: conint(ge=options.min_pulse_length, strict=True)
     pulse_sequence: conlist(conint(ge=0, strict=True), unique_items=True)

--- a/tests/experiments/experiment_unittests.py
+++ b/tests/experiments/experiment_unittests.py
@@ -287,7 +287,7 @@ def run_tests(raw_args=None, buffer=True, print_results=True):
     parser.add_argument("--experiments", required=False, nargs="+", default=None,
                         help="Only run the experiments specified after this option. Experiments \
                             specified must exist within the top-level Borealis experiments directory.")
-    parser.add_argument("--kwargs", required=False, nargs="+", default=list,
+    parser.add_argument("--kwargs", required=False, nargs="+", default=list(),
                         help="Keyword arguments to pass to the experiments. Note that kwargs are passed to all "
                              "experiments specified.")
     parser.add_argument("--module", required=False, default='__main__',
@@ -327,7 +327,7 @@ def run_tests(raw_args=None, buffer=True, print_results=True):
     if experiments is None:  # Run all unit tests and experiment tests
         build_unit_tests()
         build_experiment_tests()
-        argv = None
+        argv = [sys.argv[0]]
     else:  # Only test specified experiments
         build_experiment_tests(experiments, args.kwargs)
         exp_tests = []

